org.freeciv.Freeciv.json

```diff
@@ -1,44 +1,45 @@
 {
-    "app-id": "org.freeciv.Freeciv",
```

Comment:
please don't rewrite the whole manifest because you don't like the indentation.

Reply:
That wasn't done intentionally, it was the JSON language server auto-formatting on save in NeoVim

Reply:
indentation should be fixed now